setup.py

```diff
@@ -17,7 +17,9 @@
     'requests',
     'jsonpickle',
     'pyelliptic',
-    'ecdsa']
+    'ecdsa',
+    'collective.indexing == 2.0b1'
```

Comment:
It seems that there's no requirements.txt in zerodb itself. In zerodb-server there is, but seems could not control dependency of zerodb